.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Report a bug
+about: Let us know so we can fix it!
+---
+
+<!-- Thanks for helping us improve Cesium! Please describe what the expected behavior is vs what actually happens. --> 
+
+Operating System:
+
+Browser:
+<!-- Creating a Sandcastle example (https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/) that reproduces the issue helps us a lot in tracking down bugs -->
```

Comment:
The Sandcastle example / code example is the most important part.  The OS/browser matters, but less often so please put the code example request at the top.